iOS/MasterTimeline/MasterTimelineViewController.swift

```diff
@@ -263,8 +263,8 @@ class MasterTimelineViewController: UITableViewController, UndoableCommandRunner
 		
 		// Set up the star action
 		let starTitle = article.status.starred ?
-			NSLocalizedString("Unstar", comment: "Unstar") :
-			NSLocalizedString("Star", comment: "Star")
+			NSLocalizedString("Unstar", comment: "Accessibility label for action: Unstar") :
+			NSLocalizedString("Star", comment: "Accessibility label for action: Star")
```

Comment:
We should probably drop the “Accessibility label for action:” part of the comment parameters. These labels are not just for accessibility — they’re used as labels that display for swipe actions.

Reply:
Of course, done. My rationale when I did this was that in this specific context, those labels aren't being used for display since the swipe actions used in the iOS app only show an icon—so the comment could clarify how the text would end up being used. But that is dependent on the current implementation of the cells. Anyway, removed of course. :)